app/uk/gov/hmrc/openidconnect/userinfo/connectors/AuthConnector.scala

```diff
@@ -31,11 +31,11 @@ trait AuthConnector extends uk.gov.hmrc.play.auth.microservice.connectors.AuthCo
   val http: HttpGet
 
   def confidenceLevel()(implicit hc: HeaderCarrier): Future[Option[Int]] = {
-      http.GET(s"$authBaseUrl/auth/authority") map { resp =>
-        (resp.json \ "confidenceLevel").asOpt[Int]
-      } recover {
-        case e: Throwable => None
-      }
+    http.GET(s"$authBaseUrl/auth/authority") map { resp =>
+      (resp.json \ "confidenceLevel").asOpt[Int]
+    } recover {
+      case e: Throwable => None
+    }
```

Comment:
This is notorious in this microservice. I am not sure if I want to fix all the code :) Definitely I will fix my changes although they will not fit well the code standard anymore ;)